server/docs/hellobooks_api_doc.yaml

```diff
@@ -418,6 +418,45 @@ paths:
               schema:
                 $ref: "#/components/schemas/errorResponse"
 
+  /auth/assignrole:
+    post:
+      tags:
+        - Users
+      summary: Assign role to users
+      description: Authenticate by user role before they can perform certain actions
```

Comment:
Something doesn't seem right here... Since it is a secured route, I expected to see the required security added

Reply:
Will sort that after DEMO today.